packages/core/test/unit/sql/literal.test.ts

```diff
@@ -53,6 +56,7 @@ describe('json', () => {
       sqlite3: `json_extract(\`metadata\`,'$.pg_rating.is') = '"U"'`,
       mariadb: `json_compact(json_extract(\`metadata\`,'$.pg_rating.is')) = '"U"'`,
       mysql: `json_extract(\`metadata\`,'$.pg_rating.is') = CAST('"U"' AS JSON)`,
+      oracle: `json_value("metadata",'$."pg_rating"."is"') = 'U'`,
```

Comment:
In another file I already mentioned the quoting, that question is here as well. Should be the same cause, so I'll not mention it further in this file

Reply:
Doesn't require escaping.